charts/argo/templates/server-cluster-roles.yaml

```diff
@@ -45,6 +46,17 @@ rules:
   - events
   verbs:
   - watch
+{{- if .Values.server.sso }}
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  resourceNames:
+  - sso
+  verbs:
+  - get
+  - create
```

Comment:
The create permission cannot be attached to a specific resource name like is done, here will need to allow create on all for create permissions to support.

Reply:
fixup it allows to create all secrets

Reply:
Thanks looks like CI is failing however, I wonder if it is because the old helm repos have finally been taken down?

Reply:
I tried to fix it, but cannot find a good solution now...
- https://stackoverflow.com/a/65408163
- https://github.com/helm/chart-testing/issues/161

```sh
> docker run -v `pwd`:/workdir --rm gcr.io/kubernetes-charts-ci/test-image:v3.1.0 ct lint --debug --config .circleci/chart-testing.yaml --lint-conf .circleci/lintconf.yaml --helm-repo-extra-args ‘--stable-repo-url=https://charts.helm.sh/stable’
```

Reply:
I guess that it's time to change ci char-testing image from `gcr.io/kubernetes-charts-ci/test-image:v3.1.0` to `quay.io/helmpack/chart-testing:v3.1.0`. it means we need to upgrade ci helm version from 2.x to 3.x.

Reply:
fix it in https://github.com/argoproj/argo-helm/pull/537